src/main/java/org/sonarlint/intellij/messages/AnalysisListener.java

```diff
@@ -28,7 +28,7 @@
  * Notifies about analysis tasks starting. It will be called for any analysis task, regardless of the trigger, if it is background or not, etc.
  */
 public interface AnalysisListener {
-  Topic<AnalysisListener> TOPIC = Topic.create("SonarLint analysis start", AnalysisListener.class);
+  Topic<AnalysisListener> TOPIC = Topic.create("SonarQube for IntelliJ Analysis Start", AnalysisListener.class);
```

Comment:
Why is `Analysis Start` now uppercase? I see numerous positions where analysis and following texts are either uppercase or lowercase

Reply:
This is purely technical and not displayed anywhere (perhaps in the logs). I noticed we sometimes capitalize the topic names. Not sure what is the best practice there but in any case this is very minor IMO 😄

Reply:
I changed all topics name to lower case 👍🏻